.github/workflows/push-registry-app-files-supabase.yaml

```
@@ -0,0 +1,54 @@
+        uses: actions/setup-node@v4.0.3
+        with:
+          node-version: 18
+          registry-url: https://registry.npmjs.org/
+          cache: 'pnpm'
+      - uses: Ana06/get-changed-files@v2.3.0
+        id: changed_files
+        name: Get changed files
+        with:
+          format: json
+      - name: Upload modified or newly added *.app.mjs files to Supabase
+        uses: ./.github/actions/push-registry-app-files-supabase
```

actionlint (1.7.4)

invalid runner name "node16" at runs.using in "Push registry app files to Supabase" action defined at "/home/jailuser/git/.github/actions/push-registry-app-files-supabase". valid runners are "composite", "docker", and "node20". see https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runs

(action)